### PR TITLE
Changelog and version update

### PR DESCRIPTION
**Pull request by xflesym**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+13.0.0:
+ - Removed support for v5.
+ - Updated get_ad_group_bid_modifier example to limit number of results in query.
```
Comment (mcloonan) [inline ChangeLog:3]: We don't need to mention any of this in the changelog. The examples aren't bundled into the gem.

Comment (mcloonan) [reply ChangeLog:3]: To be clear I meant the last 3 lines of the entry. I had them highlighted but I forgot that GitHub doesn't work that way :)

Comment (xflesym) [reply ChangeLog]: Yup makes sense :) I've removed the 3 lines about examples.

**Review by mcloonan — APPROVED** [review]

**Comment by krit8669** [comment]
What is all this bc I don't get on any of it or do any of this shit my
phone hacked and all u helping

**Comment by krit8669** [comment]
I'm not subscribe to nothing my hacker is I aint

**Pull request merged**